src/writer/serve.py

```
@@ -146,6 +155,75 @@
+            f"{self.default_key_prefix}{key}",
+            value,
+            user_id=user_id
+            )
+
+    def get(self, key: str, user_id: Optional[str] = None) -> Any:
+        return self.api_client.get_value(
+            f"{self.default_key_prefix}{key}",
+            user_id=user_id
+            )
+
+    def delete(self, key: str, user_id: Optional[str] = None):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)